ims-lti.gemspec

```diff
@@ -1,16 +1,15 @@
-# coding: utf-8
 # lib = File.expand_path('../lib', __FILE__)
 # $LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
 
-require_relative "lib/ims/lti/version"
+require_relative 'lib/ims/lti/version'
 
 Gem::Specification.new do |spec|
   spec.name = 'ims-lti'
   spec.version = IMS::LTI::VERSION
   spec.authors = ['Instructure']
   spec.email = 'opensource@instructure.com'
-  spec.summary = %q{Ruby library for creating IMS LTI tool providers and consumers}
-  spec.homepage = %q{http://github.com/instructure/ims-lti}
+  spec.summary = 'Ruby library for creating IMS LTI tool providers and consumers'
+  spec.homepage = 'http://github.com/instructure/ims-lti'
```

Comment:
Since our goal is to upstream these changes, do you think the maintainer will have any issue with these changes?